src/main/java/TaskList.java

```diff
@@ -0,0 +1,82 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class TaskList {
+    private static final String indent = "    ";
+    private final List<Task> tasks;
+
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    public void addTodo(String s) {
```

Comment:
```suggestion
    public void addTodo(String userInput) {
```
The variable name ```s``` might be too short and not descriptive enough for readers.